src/Facades/StatamicNav.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Rapidez\Statamic\Facades;
+
+use Illuminate\Support\Facades\Facade;
+
+/**
+ * @see \Rapidez\Statamic\StatamicNav
+ */
+class StatamicNav extends Facade
+{
+    protected static function getFacadeAccessor()
+    {
+        return 'rapidez-statamic-nav';
```

Comment:
I suggest returning `\Rapidez\Statamic\StatamicNav::class`
Makes it easier to find the correct class by simply clicking through which is not possible if it's a basic string